README.rst

```diff
@@ -63,7 +74,7 @@ Documentation
 -------------
 
 The documentation for this this library is hosted on
-`Read The Docs <https://pywincffi.readthedocs.org/>`_.
+`ReadTheDocs <https://pywincffi.readthedocs.org/>`_.
```

Comment:
nit: Why no spaces? The homepage for it calls itself "Read The Docs", with spaces.